modules/ossm-federation-create-import.adoc

```diff
@@ -18,32 +18,32 @@ Services are imported with the name `<exported-name>.<exported-namespace>.svc.<S
 
 [NOTE]
 ====
-You can configure services for import even if they haven't been exported yet. When a service that matches the value specified in the ImportServiceSet is deployed and exported, it will be automatically imported.
+You can configure services for import even if they haven't been exported yet. When a service that matches the value specified in the ImportedServiceSet is deployed and exported, it will be automatically imported.
 ====
 
 ////
 .Procedure from the Console
 This is conjecture about what the flow might look like.
 
-Follow this procedure to create an `ImportServiceSet` with the web console. This example shows the green-mesh importing the ratings service that was exported by the red-mesh.
+Follow this procedure to create an `ImportedServiceSet` with the web console. This example shows the green-mesh importing the ratings service that was exported by the red-mesh.
 
 . Log in to the {product-title} web console as a user with the cluster-admin role.
 . Navigate to *Operators* → *Installed Operators*.
 . Click the *Project* menu and select the project where you installed the control plane for the mesh you want to import services into. For example, `green-mesh-system`.
-. Click the {SMProductName} Operator, then click *Istio Service Mesh ImportServiceSet*.
-. On the *Istio Service Mesh ImportServiceSet* tab, click *Create ImportServiceSet*.
-. On the *Create ImportServiceSet* page, click *YAML* to modify your configuration.
+. Click the {SMProductName} Operator, then click *Istio Service Mesh ImportedServiceSet*.
```

Comment:
I didn't see this section in the preview.

Reply:
It's hidden, we don't actually have a GUI yet, I was speculating what the workflow might look like if we had one.